### PR TITLE
Docker build cleanup

### LINKED ISSUE #638: Don't copy entire root when building Docker
Instead of

```
COPY . .
```

Could we just copy the relevant files & folders?

### PR DESCRIPTION
**Pull request by dnlup**
Remove `.dokerignore` file and use consistent naming in npm scripts.

Fixes #638 .

### CODE REVIEW

**Comment by dnlup** [comment]
I think the main drawback we have here js that now we are sending the whole root dir as a context before building the image:

* https://docs.docker.com/engine/reference/builder/#dockerignore-file

**Comment by ronag** [comment]
> I think the main drawback we have here js that now we are sending the whole root dir as a context before building the image:
> 
> * https://docs.docker.com/engine/reference/builder/#dockerignore-file

Good point. Maybe do both then? COPY and dockerignore?

**Comment by dnlup** [comment]
> > I think the main drawback we have here js that now we are sending the whole root dir as a context before building the image:
> > 
> > * https://docs.docker.com/engine/reference/builder/#dockerignore-file
> 
> Good point. Maybe do both then? COPY and dockerignore?

Yes. Just to be sure, we still won't `COPY . .` but each asset needed individually, right?

File: .dockerignore [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
-.clinic
-.nyc_output
-coverage
-docs
-docsify
-types
-.husky
-examples
-benchmarks
+# Ignore everything but the stuff following the `*` with the `!`
```
Comment (dnlup) [inline .dockerignore:1]: I opted for a "exclude everything" by default. It keeps the list shorter.

**Review by Ethan-Arrowood — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**